validator/client/src/main/java/tech/pegasys/teku/validator/client/DefaultValidatorStatusLogger.java

```diff
@@ -70,7 +70,7 @@ private void printValidatorStatusesOneByOne(
           Optional.ofNullable(validatorStatuses.get(publicKey));
       maybeValidatorStatus.ifPresentOrElse(
           validatorStatus ->
-              STATUS_LOG.validatorStatus(validatorStatus.name(), publicKey.toAbbreviatedString()),
+              STATUS_LOG.validatorStatus(publicKey.toAbbreviatedString(), validatorStatus.name()),
```

Comment:
I think the problem is actually in `StatusLogger`: https://github.com/ConsenSys/teku/blob/f38689facd9ec15aa74065ad2462be3488d4e361/infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java#L255-L257